layers/fpm/bref.ini

```diff
@@ -10,6 +10,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 memory_limit=10240M
```

Comment:
TODO remove that file after merging everything we can into the main php.ini file